templates/index.html

```diff
@@ -42,7 +42,7 @@ <h4>Browser Configuration</h4>
       <div class="row single">
         <div class="left">
           <h4>32-bit Support</h4>
-          <p>Does your distribution needs 32bit support. <br> Default is 64bit.</p>
+          <p>Does your distribution need 32bit support? <br>The Default is 64bit.</p>
```

Comment:
“The” is not required here
Other changes seems fine

Reply:
@pranav1698  The is added.